LICENSE

```diff
@@ -0,0 +1,668 @@
+GNU Affero General Public License version 3 – OpenPaaS
```

Comment:
Is it compatible with gatling license ?
Do we really want AGPL for this kind of tool ?